audb/core/load.py

```diff
@@ -1037,6 +1037,116 @@ def load(
     return db
 
 
+def load_attachment(
+        name: str,
+        attachment: str,
+        *,
+        version: str = None,
+        cache_root: str = None,
+        num_workers: typing.Optional[int] = 1,
```

Comment:
As you already pointed out before, a `num_workers` argument doesn't really make sense since we will anyway only load one archive at max.

Reply:
I removed the `num_workers` argument.